src/main.rs

```diff
@@ -99,6 +99,13 @@ struct Opt {
     /// quoting.
     #[structopt(value_name = "CHAR", long = "quote", default_value = "\"")]
     quote: CharSpecifier,
+
+    /// Should a null-byte (\x0) be interpreted as an empty cell
+    /// this option exists because it is not always possible to pass
+    /// a null byte into --null on the command line.
+    /// https://github.com/faradayio/scrubcsv/issues/17
+    #[structopt(long = "null-byte-as-empty")]
+    null_byte_as_empty: bool,
```

Comment:
Before adding an option, did we try `--null '^\x00$'? Rust's `regex` library allows bytes to be escaped.

If we need to add another null option besides `--null`, we should aim for a consistent naming convention, both with `-null` and the other existing options. Most of our existing options are either "--verb" or "--noun", and `--null-byte-as-empty` . It's definitely good to _start_ with `--null-` like you do here, because that will ensure the right sort order on the display.